src/activity-logger/background/log-page-visit.test.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-env jest */
+/* eslint import/namespace: "off" */
+
+import maybeLogPageVisit from 'src/activity-logger/background/log-page-visit'
+import * as storePage from 'src/page-storage/store-page'
+import db from 'src/pouchdb'
+
+afterAll(async () => {
+    await db.destroy()
+})
+
+describe('maybeLogPageVisit', () => {
```

Comment:
I would not recommend to use rewire because of the [limitations](https://github.com/jhnns/rewire#limitations). Also it is implemented for CommonJS which is not future proof for when ES modules are supported natively.